keel-core/src/main/kotlin/com/netflix/spinnaker/keel/actuation/ResourceActuator.kt

```diff
@@ -173,6 +176,7 @@ class ResourceActuator(
             plugin.create(resource, diff)
               .also { tasks ->
                 publisher.publishEvent(ResourceActuationLaunched(resource, plugin.name, tasks, clock))
+                diffFingerprintRepository.markActionTaken(id)
```

Comment:
this is a change. previously, we cared only about whether we saw a diff. now, we're marking that we are taking action on a diff. this is used in the UnhappyVeto, so that we only veto an artifact if we've taken action on it the allotted number of times.

no behavior change yet

Reply:
This feels like the wrong repository/table to use to store this information. IMHO, this should go in the `resources` table, as we're counting action taken to resolve a diff from desired to current state of the resource. It's got nothing to do with diff fingerprints.

Reply:
@emjburns doesn't the column `count` in `diff_fingerprint` table is the counter for each time we are taking action on a diff? or is it just when we are seeing a diff?

Reply:
The existing column is just when we see a diff

Reply:
I think you could make a case for either place. It is super related to diff fingerprints because we take action on the diff that that fingerprint represents. I think you could also make the case that we should store the fingerprint with the resource. But, it seems much simpler to store them in a different table. To me the resource table is about the config not about what we're doing in the moment.

I'd like to leave it the way it is.